xpti/CMakeLists.txt

```diff
@@ -1,17 +1,28 @@
-cmake_minimum_required(VERSION 2.8.9)
+cmake_minimum_required(VERSION 3.8)
```

Comment:
The LLVM minimum version is 3.13.4, I think. Is there a reason that 3.8 is required here?

Reply:
XPTI is designed to be buildable (and usable) outside of LLVM tree, and `3.8` is what XPTIFW project uses by default, so I just raised this version to match the framework.